src/Microsoft.Azure.SignalR.Management/ServiceHubContextBuilder.cs

```diff
@@ -49,13 +50,20 @@ internal async Task<ServiceHubContext> CreateAsync(string hubName, CancellationT
             }
             catch
             {
+                shouldDispose = true;
                 if (host != null)
                 {
                     await host.StopAsync();
```

Comment:
using here?
```cs
using (host) {
    await host.StopAsync();
}
```

Reply:
Got it. `using host;` is forbidden, but `using(host)` is Ok.